src/code_16c8e0.c

```diff
@@ -209,7 +215,70 @@ INCLUDE_ASM(s32, "code_16c8e0", func_802409F4);
 
 INCLUDE_ASM(s32, "code_16c8e0", func_80240AA8);
 
-INCLUDE_ASM(s32, "code_16c8e0", delete_actor);
+void delete_actor(Actor *actor) {
+    ActorPart* partsTable;
+    ActorPart* actorPartTemp;
+    BattleStatus* battleStatus;
+    s32 i;
+
+    // TODO hard-coded
```

Comment:
Yeah, we'll need a define for this that is used in the struct Actor decorationList